cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/OpenSamlConfigBean.java

```diff
@@ -62,6 +65,7 @@ public ParserPool getParserPool() {
     @PostConstruct
     public void init() {
         LOGGER.debug("Initializing OpenSaml configuration...");
+        Assert.notNull(this.parserPool, "parserPool cannot be null");
```

Comment:
True. Probably not needed, but annotations only kick it if there's a validator process. If you are calling `init` manually..., then they dont. Though, I'd agree that this is probably overkill.